README.md

```diff
@@ -41,6 +41,16 @@ All requirements are installed as part of `pip install` or if you're planning on
 * schema
 * yara-python
 
+**Note: Windows users will need to have [Microsoft Visual C++ 14.0 Build Tools](https://visualstudio.microsoft.com/visual-cpp-build-tools) installed in order to compile yara-python.**
```

Comment:
Consider having one "Note" with two bullet points below, one for Windows and one for Linux (the concern is too many notes can create note-blindness)

**Note:**
- **Windows** users will need to have...
- **Linux** users will need to have...

- 

- 

-